governance/governance.go

```diff
@@ -29,8 +29,6 @@ import (
 // the max time we'll let a contract remain unconfigured by the provider
 const MAX_CONTRACT_UNCONFIGURED_TIME_M = 20
 
-const MAX_CONTRACT_PRELAUNCH_TIME_M = 10
-
 const MAX_MICROPAYMENT_UNPAID_RUN_DURATION_M = 60
 
 // enforced only after the workloads are running
```

Comment:
I just added this in the issue. Please remove MAX_CONTRACT_UNCONFIGURED_TIME_M,  MAX_MICROPAYMENT_UNPAID_RUN_DURATION_M, MAX_AGREEMENT_ACCEPTANCE_WAIT_TIME_M too because they are not refrenced anywhere.